chore: uninstall deprecated color module

lazysoundsystem opened the pull request:
Refs: OPS-8561

Color module is now deprecated.

Should probably revise the versions of the other modules too, but that can be in another ticket.

attiks commented:
```
Pulling drupal (unocha/starterkit-site:local)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.
Continue with the new image? [yN]
```

lazysoundsystem commented:
The problem was that composer has updated and changed policy since we last built this repo - it now needs 'allow-plugins' authorizations https://getcomposer.org/doc/06-config.md#allow-plugins . I've added that and some other updates in https://github.com/UN-OCHA/drupal-starterkit/pull/3 and the Makefile should complete now.
